app/views/users/edit.html.erb

```diff
@@ -1,6 +1,12 @@
+<% content_for :navbar_links do %>
+  <%= render 'layouts/basic_navbar/user_links' %>
+<% end %>
+
+<%= render "layouts/page_heading", locals = {heading: "Edit Profile: #{current_user.nickname"}%>
```

Comment:
My thinking was that 

mhecker is making a new business
you are editing the profile of mhecker

I'll change it to be Title: username in both cases

for consistency